webapp/channels/src/components/channel_invite_modal/channel_invite_modal.tsx

```diff
@@ -408,7 +408,7 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
                 valueWithImage={true}
                 backButtonText={localizeMessage('multiselect.cancel', 'Cancel')}
                 backButtonClick={closeMembersInviteModal}
-                backButtonClass={'btn-cancel tertiary-button'}
+                backButtonClass={'btn-tertiary tertiary-button'}
```

Comment:
This is likely outside the scope of this PR, but I was checking if we still use this `tertiary-button` class to see if we still use it, and it looks like there's still a few spots we do. I also noticed that we have a `tertiary-button` mixin in the CSS that's used in a bunch of places.

Is that something we should also remove eventually?

Reply:
Yes, we will remove it, but since some of those things are used elsewhere, I didn't wanted to remove everything right now, in the interest of keeping the scope limited to whatever extent.